base_layer/chat_ffi/Cargo.toml

```diff
@@ -11,7 +11,6 @@ minotari_app_utilities = { path = "../../applications/minotari_app_utilities" }
 tari_chat_client = { path = "../contacts/examples/chat_client" }
 tari_common = { path = "../../common" }
 tari_common_types = { path = "../common_types" }
-tari_comms = { path = "../../comms/core" }
```

Comment:
this correct?

Reply:
I was surprised to see this as well. Machete caught it this morning (not yesterday), so it may have to do with the last PR. Everything compiles though. And nothing in comms was in the build.rs.

Reply:
Could also be when I shuffled around the imports I selected an import from an existing lib that was re-exporting whatever was used from comms. The chat client lib does re-export a few things and I know I changed an import or two.